packages/client/src/utils/constants.ts

```diff
@@ -58,6 +58,7 @@ export const ROLE_REGISTRATION_AGENT = 'REGISTRATION_AGENT'
 export const ROLE_LOCAL_REGISTRAR = 'LOCAL_REGISTRAR'
 export const FIELD_AGENT_ROLES = [ROLE_FIELD_AGENT]
 export const SYS_ADMIN_ROLES = ['LOCAL_SYSTEM_ADMIN']
+export const NATL_ADMIN_ROLES = 'NATIONAL_SYSTEM_ADMIN'
```

Comment:
As it is called roles so `['NATIONAL_SYSTEM_ADMIN']` seems more appropriate

Reply:
I agreee